src/components/SuccessContent.tsx

```diff
@@ -35,13 +57,35 @@ const SuccessContent = ({ title, email }) => {
       <Typography variant="body1">
         {t(AUTH.SIGN_IN_SUCCESS_EMAIL_PROBLEM)}
       </Typography>
+      <br />
+      <Stack direction="row" justifyContent="center" spacing={1}>
+        <Button
+          variant="text"
+          color="primary"
+          id={BACK_BUTTON_ID}
+          onClick={handleBackButtonClick}
+        >
+          {t(AUTH.BACK_BUTTON)}
+        </Button>
+        <Button
+          variant="outlined"
+          color="primary"
+          onClick={onClickResendEmail}
+          id={RESEND_EMAIL_BUTTON_ID}
+          disabled={clicked}
+        >
+          {t(AUTH.RESEND_EMAIL_BUTTON)}
+        </Button>
+      </Stack>
     </Container>
   );
 };
 
 SuccessContent.propTypes = {
   title: propTypes.string.isRequired,
   email: propTypes.string.isRequired,
+  handleBackButtonClick: propTypes.func,
+  handleResendEmail: propTypes.func,
 };
```

Comment:
Actually you don't need `propTypes` if you have a tsx file. Instead use `type Props` from typescript!